internetarchive/cli/ia_tasks.py

```diff
@@ -96,28 +94,25 @@ def main(argv, session):
                                 data=data)
         j = r.json()
         if j.get('success'):
-            print('success: {}'.format(j.get('value', dict()).get('log')))
+            print(f'success: {j.get("value", {}).get("log")}')
```

Comment:
This is a case where I feel the new syntax stretches a bit far. While the previous version was not any clearer, it maybe should have been written differently in the first place and we should decide whether we want such "complex" statements within the format strings.

Reply:
Yeah, I thought about this one a bit while writing it. The alternative would be to put it in a variable which is then used in the f-string. I'm not really sure that's better though.

Reply:
Yeah, this definitely should've been written differently in the first place, IMO. I think using a variable would be preferable, I'd probably do something like this:

```
task_log_url = j.get("value", {}).get("log")
print(f'success: {task_log_url}')
```

The variable name, `task_log_url`, at least gives some idea what is being printed here. No worries if you don't deal with this in this PR. I can fix it after merging.